package/setup.py

```diff
@@ -567,112 +560,20 @@ def long_description(readme):
         # (LONG_DESCRIPTION is not really needed)
         LONG_DESCRIPTION = "MDAnalysis -- https://www.mdanalysis.org/"
 
-    CLASSIFIERS = [
-        'Development Status :: 6 - Mature',
-        'Environment :: Console',
-        'Intended Audience :: Science/Research',
-        'License :: OSI Approved :: GNU General Public License v3 or later (GPLv3+)',
-        'Operating System :: POSIX',
-        'Operating System :: MacOS :: MacOS X',
-        'Operating System :: Microsoft :: Windows ',
-        'Programming Language :: Python',
-        'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.9',
-        'Programming Language :: Python :: 3.10',
-        'Programming Language :: Python :: 3.11',
-        'Programming Language :: Python :: 3.12',
-        'Programming Language :: C',
-        'Topic :: Scientific/Engineering',
-        'Topic :: Scientific/Engineering :: Bio-Informatics',
-        'Topic :: Scientific/Engineering :: Chemistry',
-        'Topic :: Software Development :: Libraries :: Python Modules',
-    ]
     config = Config()
     exts, cythonfiles = extensions(config)
 
-    install_requires = [
-          'numpy>=1.23.2',
-          'GridDataFormats>=0.4.0',
-          'mmtf-python>=1.0.0',
-          'joblib>=0.12',
-          'scipy>=1.5.0',
-          'matplotlib>=1.5.1',
-          'tqdm>=4.43.0',
-          'threadpoolctl',
-          'packaging',
-          'fasteners',
-          'mda-xdrlib',
-          'waterdynamics',
-          'pathsimanalysis',
-    ]
-
     setup(name='MDAnalysis',
           version=RELEASE,
-          description='An object-oriented toolkit to analyze molecular dynamics trajectories.',
           long_description=LONG_DESCRIPTION,
           long_description_content_type='text/x-rst',
```

Comment:
Should this be removed in favour of
```
readme = {file = "README", content-type = "text/x-rst"}
```

I haven't double checked the documentation... I'm just comparing with https://github.com/MDAnalysis/mdanalysis/pull/4463, so feel free to discard if this is a stupid question.

Reply:
The issue here is that the readme exists outside of the package, so the pyproject.toml approach won't work - i.e. when you have an sdist your readme target will be a non existent file.

We could long term fix this with some symlinks or just a pre packaging copy step, but probably best not to do so here.